foundry/main.rs

```diff
@@ -60,14 +62,46 @@ fn run() -> Result<(), String> {
         .about("Foundry client")
         .subcommand(SubCommand::with_name("commit-hash").about("Print the commit hash of the source tree"));
 
+    let app = app.arg(
+        clap::Arg::with_name("module_arguments")
+            .short("D")
+            .help("module arguments. For example  ''")
+            .takes_value(true)
+            .multiple(true),
+    );
```

Comment:
@dynaxis I tried to add `-D` in Config struct, but failed.
`structconf` does not support multiple arguments